libs/ledgerjs/packages/hw-app-aptos/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "@ledgerhq/hw-app-aptos",
+  "version": "6.29.4",
+  "description": "Ledger Hardware Wallet Aptos Application API",
+  "keywords": [
+    "Ledger",
+    "LedgerWallet",
+    "algo",
```

remove this one please

```suggestion
    "algo",
```